scripts/mbedtls_dev/bignum_common.py

```diff
@@ -267,6 +296,12 @@ def boundary(self) -> int:
     def arg_n(self) -> str:
         return self.format_arg(self.val_n)
 
+    def format_arg(self, val: str) -> str:
+        if self.input_style == "variable":
+            return val.zfill(len(hex(self.int_n)) - 2)
```

Comment:
The point is that in modular operations, by default, all operands should be padded to the same size as the modulus. It may be possible to use `self.hex_digits` and override `self.hex_digits` in `ModOperationCommon`: that may be a more desirable result, but it doesn't seem simpler to achieve.